NOLF2/ClientShellDLL/ClientShellShared/GameSettings.cpp

```diff
@@ -200,7 +200,7 @@ void CGameSettings::ImplementMouseSensitivity()
 			float fScaleIncrement = g_vtMouseScaleInc.GetFloat();
 			auto szSensitivity = std::to_string(fBaseScale + (nMouseSensitivity * fScaleIncrement));
 
-			auto strConsole = std::make_unique<char[]>(devLen+std::max(xLen,yLen)+szSensitivity.length() + 14);
+			auto strConsole = std::make_unique<char[]>(devLen+std::max<int>(xLen,yLen)+szSensitivity.length() + 14);
```

Comment:
hmm... I thought with c++17 CTAD was a thing.... so maybe not in MSVC (VS2017)?

Reply:
Not sure what CTAD is. Without the `<int>`, VS2019 complains that it "expected an identifier". This is probably because `max` is a defined macro in `minwindef.h` and causes some sort of syntax error with the rest of the line.

Reply:
*TAD
CTAD is Class Template Argument Deduction

Reply:
since both `xLen` and `yLen` are `size_t` which is the return type from `strlen` 
that would be better to use.

Reply:
Done.